MantidQt/CustomInterfaces/test/ReflSettingsTabPresenterTest.h

```diff
@@ -166,6 +169,8 @@ class ReflSettingsTabPresenterTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(optionsVec[15], "ScaleFactor=2");
     TS_ASSERT_EQUALS(optionsVec[16], "MomentumTransferStep=-0.02");
     TS_ASSERT_EQUALS(optionsVec[17], "ProcessingInstructions=\"3,4\"");
+    TS_ASSERT_EQUALS(optionsVec[18], "FirstTransmissionRun=TRANS_INTER00013463.nxs");
+    TS_ASSERT_EQUALS(optionsVec[19], "SecondTransmissionRun=TRANS_INTER00013464.nxs");
```

Comment:
In addition to this, we should test that workspaces were created in the ADS, something like:

```
TS_ASSERT(AnalysisDataService::Instance().doesExist("TRANS_13464");
TS_ASSERT(AnalysisDataService::Instance().doesExist("TRANS_13464");
```

We should also clear the ADS by adding:

```
AnalysisDataService::Instance().clear();
```